drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -101,6 +101,8 @@
 #define _EXT_COMPRESSED_RGB_BPTC_SIGNED_FLOAT 0x8E8E
 #define _EXT_COMPRESSED_RGB_BPTC_UNSIGNED_FLOAT 0x8E8F
 
+#define _GL_TEXTURE_EXTERNAL_OES 0x8D65
+
```

Comment:
GLES 3 is going to be removed when the Vulkan branch is merged after 3.2's release.

Are external textures something that could be used with Vulkan too, or would it be a GLES 2 only feature?

Reply:
This is pretty core to a number of features on Android so I'm assuming there will be a vulkan equivalent.